tests/primordial_chem.in

```diff
@@ -0,0 +1,76 @@
+# copy of Cooling.in
+# *****************************************************************
+# Problem size and geometry
+# *****************************************************************
+geometry.prob_lo     =  0.0  0.0  0.0 
+geometry.prob_hi     =  1  1  1
+geometry.is_periodic =  0    0    0
+
+# *****************************************************************
+# VERBOSITY
+# *****************************************************************
+amr.v               = 0     # verbosity in Amr
+
+# *****************************************************************
+# Resolution and refinement
+# *****************************************************************
+amr.n_cell          = 1 1 1
+amr.max_level       = 0     # number of levels = max_level + 1
+amr.blocking_factor = 1   # grid size must be divisible by this
+amr.max_grid_size   = 1
+
+do_reflux = 0
+do_subcycle = 0
+
+unit_test.enabled = 1
+unit_test.temperature = 1e2
+
+unit_test.small_temp = 1.e1
+unit_test.small_dens = 1.e-60
+uni_test.max_dens_primordialchem = 3e-6
```

Comment:
This should be applicable to anyone who is using the chemistry module. It might make sense to rename this to be `chemistry.max_density_allowed` or something like that.

Reply:
It's originally defined in Microphysics, which uses namespace `unit_test`

Reply:
This is only applicable to anyone using the chemistry module by calling microphysics, so in that sense it comes from the same family of parameters like `small_temp` and `small_dens` defined just above it

Reply:
I don't think it makes sense to label this family of parameters `unit_test`. Something like `chemistry` or `primordial_chemistry` would make more sense.

Reply:
We didn't choose to call it `unit_test`, it was what was present in Microphysics. All the unit tests in Microphysics use the namespace `unit_test`

Reply:
Yes, but we can change it to whatever we want.

Reply:
namespace `unit_test` is defined in `Microphysics/unit_test/_parameters
So, you are saying I can define a new namespace (e.g., `primordial_chem`) in `Microphysics/unit_test/burn_cell_primordial_chem/_parameters`, and it will be read in just like `unit_test`?

Reply:
No, that shouldn't be necessary. You can just do `amrex::ParmParse pp("primordial_chem");` and then read in the values `pp.query("max_density_allowed", max_density);` which will read in the value `primordial_chem.max_density_allowed` into the variable `max_density`.

Reply:
I tried that before but it didn't work for some reason. I will try it again. Where should it go btw? In `Chemistry.hpp` or `test_primordial_chem.cpp` (I guess we only want to read it once, so it could go in `preCalculateInitialConditions`, but then would `Chemistry.hpp` know about it?)?

Reply:
Also, we have another parameter: `unit_test.enabled` which when set to 1, enables primordial chem. Ideally we would want this to be called `primordial_chem.enabled`. This is parmparsed and used in `RadhydroSimulation.hpp`.

Reply:
Where is the code that currently reads in these parameters?

Reply:
`unit_test.enabled` is read in `RadhydroSimulation.hpp`, in `void RadhydroSimulation<problem_t>::readParmParse()`

Reply:
Resolved by using `Simulationdata`